flink/v1.19/flink/src/main/java/org/apache/iceberg/flink/sink/shuffle/DataStatisticsCoordinator.java

```diff
@@ -71,6 +76,7 @@ class DataStatisticsCoordinator implements OperatorCoordinator {
   private transient boolean started;
   private transient AggregatedStatisticsTracker aggregatedStatisticsTracker;
   private transient AggregatedStatistics completedStatistics;
+  private transient AggregatedStatistics globalStatistics;
```

Comment:
for sketch statistics type, `globalStatistics` only stores range bounds (a much smaller array)